### PR TITLE
Covidata: adding tests

### PR DESCRIPTION
**Pull request by Jeddaa**
-  connect to an api to:
  - display countries affected
  - display number of people infected
  - display provinces
  - display casualties
- Style the app
- write tests for components

- demo link: https://www.loom.com/share/9e0ccf5cd6ca4433b55a45ab967e559a
- deployed app link: https://covid-ata.netlify.app/

### CODE REVIEW

File: src/components/__test__/Component.tes.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// import { render, screen } from '@testing-library/react';
+// //
+
+// test('renders the home page', () => {});
```
Comment (andersonlebon) [inline src/components/__test__/Component.tes.js]: image

- [ ] Nice work so far your app is working well however  yous till need tp work on the design of you project to look like the template. kindly make sure the colloer choosen is on of the template one, the design should look exaxtly like the  [template design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)).

Your design | the template
---|---
image | image image

Comment (andersonlebon) [inline src/components/__test__/Component.tes.js]: - [ ] Kindly clear you code by removing commented code or unnecessary files.

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @Jeddaa 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

almost-there

## Suggested changes
_Check the comments unde


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @Jeddaa , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Well done implementing the requested changes. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**